src/howto/create-project/index.md

```diff
@@ -0,0 +1,121 @@
+# Create a new project
+[[toc]]
+
+There are several methods of creating a new <MainPlatformName /> project:
+- [<MobileAppName /> ](./index.md#create-a-project-in-mergin-maps-input) offers the quickest (albeit limited) way of creating a <MainPlatformName /> project.
```

Comment:
can you please double-check in generated version that the `MobileAppName` expands when used as title inside the link?

Reply:
This seems to work.